prost-build/src/lib.rs

```diff
@@ -1115,6 +1115,26 @@ impl Module {
     }
 }
 
+impl fmt::Debug for Module {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        fmt::Display::fmt(self, f)
+    }
+}
```

Comment:
Why not derive this?

Reply:
That's a good point. Let me step that back one and have it derive instead.

Reply:
Updated. This was a relic of my own wrapper class where I _couldn't_ derive `Debug` because `Module` didn't derive it either. `Display` impl is also corrected too.